docs/pages/connect/ssh.md

```
@@ -0,0 +1,66 @@
+
+```console
+$ rsync -avz myapp@ssh.statespace.com:./data/ ./backup/
+```
+
+## Run commands
+
++Execute a single command without an interactive session:
+
+```console
+$ ssh myapp@ssh.statespace.com "cat README.md"
+```
```

markdownlint-cli2 (0.21.0)

[warning] Files should end with a single newline character

(MD047, single-trailing-newline)